.github/workflows/auto_pr.yml

```diff
@@ -0,0 +1,82 @@
+# Make sure you have the secrets MACHINE_USER_PAT and MACHINE_USER_FORKED_REPOSITORY set
+
+name: Create official images PR
+
+on:
+  push:
+    branches:
+      - master
+
+    paths:
+      - "generate-stackbrew-*.sh"
+      - "**/Dockerfile"
+      - "**/architectures"
+
+jobs:
+  auto-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the docker-node repo
+        uses: actions/checkout@v2
+        with:
+          path: docker-node
+
+      - name: Checkout the official-images repo
+        uses: actions/checkout@v2
+        with:
+          path: official-images
+          repository: docker-library/official-images
+
+      - name: Generate PR text
+        uses: actions/github-script@v3
+        id: pr-format
+        with:
+          script: |
+            const basePath = `${process.env.GITHUB_WORKSPACE}/docker-node`;
+            const script = require(`${basePath}/.github/prFormat.js`);
+            return script({ github, context, maintainersFile: `${basePath}/ACTIVE_MAINTAINERS` });
```

Comment:
Think we could even just drop the ACTIVE_MAINTAINERS and just hardcode the list. AFAIK, it's only used for the PR flow